Src/CSharpier.Tests/FormattingTests/TestFiles/cs/CollectionExpressions.test

```diff
@@ -1,6 +1,8 @@
-int[] a =  [ 1, 2, 3, 4, 5, 6, 7, 8 ];
```

Comment:
I had a test case with the two spaces here and didn't even notice
: (

Reply:
If it's any consolation, I didn't notice in the code review either.